fuel-moshi/src/test/kotlin/com/github/kittinunf/fuel/FuelMoshiTest.kt

```diff
@@ -0,0 +1,114 @@
+package com.github.kittinunf.fuel
+
+import com.github.kittinunf.fuel.core.*
+import com.github.kittinunf.fuel.moshi.moshiDeserializerOf
+import com.github.kittinunf.fuel.moshi.responseObject
+import com.github.kittinunf.result.Result
+import org.hamcrest.CoreMatchers
+import org.junit.Assert
+import org.junit.Test
+
+class FuelMoshiTest {
+
+    init {
+        FuelManager.instance.basePath = "https://httpbin.org"
+
+        Fuel.testMode {
+            timeout = 15000
+        }
+    }
+
+    data class HttpBinUserAgentModel(var userAgent: String = "")
+
+    @Test
+    fun gsonTestResponseObject() {
```

Comment:
Should we name all test cases here with `moshi`?

Reply:
Oh I forgot.... Will fix immediately.